<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>td_form</class>
 <widget class="QDialog" name="td_form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1006</width>
    <height>712</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QLabel" name="label">
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);
font: 16pt &quot;Segoe UI&quot;;
</string>
     </property>
     <property name="text">
      <string>Fill out the following form</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::Shape::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QWidget" name="widget" native="true">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);
font: 10pt &quot;Segoe UI&quot;;
border-colour: 2px solid rgb(0, 0, 0);
</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QWidget" name="widget_2" native="true">
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="font">
               <font>
                <family>Segoe UI</family>
                <pointsize>12</pointsize>
                <italic>false</italic>
                <bold>false</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color: rgb(0, 0, 0);
font: 12pt &quot;Segoe UI&quot;;</string>
              </property>
              <property name="text">
               <string>Name</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="td_name"/>
            </item>
            <item>
             <widget class="QLabel" name="label_8">
              <property name="font">
               <font>
                <family>Segoe UI</family>
                <pointsize>12</pointsize>
                <italic>false</italic>
                <bold>false</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color: rgb(0, 0, 0);
font: 12pt &quot;Segoe UI&quot;;</string>
              </property>
              <property name="text">
               <string>ID</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="td_id"/>
            </item>
            <item>
             <widget class="QLabel" name="label_4">
              <property name="font">
               <font>
                <family>Segoe UI</family>
                <pointsize>12</pointsize>
                <italic>false</italic>
                <bold>false</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color: rgb(0, 0, 0);
font: 12pt &quot;Segoe UI&quot;;</string>
              </property>
              <property name="text">
               <string>Email</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="td_email"/>
            </item>
            <item>
             <widget class="QListWidget" name="listWidget">
              <property name="styleSheet">
               <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(130, 180, 255);
background-color: rgb(230, 228, 255);
border-radius: 25%
</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_2">
        <property name="styleSheet">
         <string notr="true">QRadioButton {
    color: #333333;
    font-size: 14px;
    spacing: 10px;
}

QRadioButton:checked {
    color: #0078d7;
    font-weight: bold;
}

QRadioButton::indicator {
    width: 18px;
    height: 18px;
    border-radius: 11px;
    border: 2px solid #999999;
    background-color: #ffffff;
}

QRadioButton::indicator:hover {
    border: 2px solid #0078d7;
}

QRadioButton::indicator:checked {
    background-color: #0078d7;
    border: 2px solid #005a9e;
}</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::Shape::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Shadow::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="font">
            <font>
             <family>Segoe UI</family>
             <pointsize>11</pointsize>
             <italic>false</italic>
             <fontweight>DemiBold</fontweight>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(0, 0, 0);
font: 600 11pt &quot;Segoe UI&quot;;
</string>
           </property>
           <property name="text">
            <string>Add your assigned classes with repective subject</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_5">
           <property name="font">
            <font>
             <family>Segoe UI</family>
             <pointsize>12</pointsize>
             <italic>false</italic>
             <bold>false</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(0, 0, 0);
font: 12pt &quot;Segoe UI&quot;;</string>
           </property>
           <property name="text">
            <string>Department</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox">
           <property name="styleSheet">
            <string notr="true">color: rgb(0, 0, 0);</string>
           </property>
           <property name="title">
            <string/>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QRadioButton" name="radioScience">
              <property name="text">
               <string>Science</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radioCommerce">
              <property name="text">
               <string>Commerce</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_6">
           <property name="font">
            <font>
             <family>Segoe UI</family>
             <pointsize>12</pointsize>
             <italic>false</italic>
             <bold>false</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(0, 0, 0);
font: 12pt &quot;Segoe UI&quot;;</string>
           </property>
           <property name="text">
            <string>Subjects</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_4" native="true">
           <property name="styleSheet">
            <string notr="true">color: rgb(0, 0, 0);</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QGroupBox" name="groupBox_5">
              <property name="styleSheet">
               <string notr="true">color: rgb(0, 0, 0);</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_5">
               <item>
                <widget class="QRadioButton" name="radioBusiness">
                 <property name="text">
                  <string>Business</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="radioAccountancy">
                 <property name="text">
                  <string>Accountancy</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="radioFinance">
                 <property name="text">
                  <string>Finance</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="radioEconomics">
                 <property name="text">
                  <string>Economics</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="Sciencesubjects">
              <property name="styleSheet">
               <string notr="true">color: rgb(0, 0, 0);</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_6">
               <item>
                <widget class="QRadioButton" name="radioPhysics">
                 <property name="text">
                  <string>Physics</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="radioChemistry">
                 <property name="text">
                  <string>Chemistry</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="radioMath">
                 <property name="text">
                  <string>Math</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="radioBiology">
                 <property name="text">
                  <string>Biology</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_7">
           <property name="font">
            <font>
             <family>Segoe UI</family>
             <pointsize>12</pointsize>
             <italic>false</italic>
             <bold>false</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(0, 0, 0);
font: 12pt &quot;Segoe UI&quot;;</string>
           </property>
           <property name="text">
            <string>Class</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_3">
           <property name="styleSheet">
            <string notr="true">color: rgb(0, 0, 0);</string>
           </property>
           <property name="title">
            <string/>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QRadioButton" name="radio11">
              <property name="text">
               <string>11</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radio12">
              <property name="text">
               <string>12</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_9">
           <property name="font">
            <font>
             <family>Segoe UI</family>
             <pointsize>12</pointsize>
             <italic>false</italic>
             <bold>false</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(0, 0, 0);
font: 12pt &quot;Segoe UI&quot;;</string>
           </property>
           <property name="text">
            <string>Sections</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_2">
           <property name="styleSheet">
            <string notr="true">color: rgb(0, 0, 0);</string>
           </property>
           <property name="title">
            <string/>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QRadioButton" name="radioA">
              <property name="text">
               <string>A</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radioB">
              <property name="text">
               <string>B</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="add_to_schedule">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 115, 152);
color: rgb(0, 0, 0);
font: 600 12pt &quot;Segoe UI&quot;;</string>
           </property>
           <property name="text">
            <string>Add to your schedule</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="save">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(221, 255, 254);
background-color: rgb(190, 255, 216);
color: rgb(0, 0, 0);
font: 600 14pt &quot;Segoe UI&quot;;</string>
     </property>
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
